### PR TITLE
DOCS-2249 API Discovery from all-in-one installer

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/about-wallarm/api-discovery.md [hunk 1]
```diff
@@ -262,11 +262,16 @@ Other data (endpoint values, request methods, and parameter names) is not hashed
...
+
+* Wallarm [all-in-one installer](../installation/nginx/all-in-one.md)
+* All other [forms](../installation/supported-deployment-options.md) of the Wallarm node installation as the `wallarm-appstructure` package, except for the Debian 11.x and Ubuntu 22.04 istallations
+
+During node deployment, it installs the API Discovery module but keeps it disabled by default.
 
 To enable and run API Discovery correctly:
 
-1. Make sure your Wallarm node is of the [supported version](../updating-migrating/versioning-policy.md#version-list).
+1. (Skip this step, if you are using all-in-one installer) Make sure your Wallarm node is of the [supported version](../updating-migrating/versioning-policy.md#version-list).
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md:274]: why do they need to skip the step?

File: docs/latest/about-wallarm/api-discovery.md [hunk 2]
```diff
@@ -262,11 +262,16 @@ Other data (endpoint values, request methods, and parameter names) is not hashed
 
 ## Enabling and configuring API Discovery
 
-The `wallarm-appstructure` package is included in all [forms](../installation/supported-deployment-options.md) of the Wallarm node except for the Debian 11.x and Ubuntu 22.04 packages. During node deployment, it installs the API Discovery module but keeps it disabled by default.
+API Discovery is included in:
+
+* Wallarm [all-in-one installer](../installation/nginx/all-in-one.md)
+* All other [forms](../installation/supported-deployment-options.md) of the Wallarm node installation as the `wallarm-appstructure` package, except for the Debian 11.x and Ubuntu 22.04 istallations
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md:268]: except for the Debian 11.x and Ubuntu 22.04 istallations - it is truth only for specific artifacts

File: docs/latest/about-wallarm/api-discovery.md [hunk 3]
```diff
@@ -262,11 +262,16 @@ Other data (endpoint values, request methods, and parameter names) is not hashed
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md:262]: under this task, you also wanted to update the document with all artifact versions

File: docs/latest/about-wallarm/api-discovery.md (hunk 1 above)
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md:274]: only fir indiv packages

File: docs/latest/about-wallarm/api-discovery.md (hunk 2 above)
Comment (AnastasiaTWW) [reply docs/latest/about-wallarm/api-discovery.md:268]: just say: all forms except for indiv packages for specific OS

File: docs/latest/about-wallarm/api-discovery.md (hunk 3 above)
Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-discovery.md:262]: We have discussed that: it will be included as soon as the artifacts are actually published.

**Pull request closed without merging**